core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -4286,7 +4286,7 @@ public void heartbeat() throws InterruptedException {
         long jobId = job.getId();
         JobMasterClient client = mJobMasterClientPool.acquire();
         try {
-          alluxio.job.wire.JobInfo jobInfo = client.getStatus(jobId);
+          alluxio.job.wire.PlanInfo jobInfo = client.getStatus(jobId);
```

Comment:
Naming thing, it feels kind of weird to call this a `plan` when everything is else is a `job`, ie. `job service`, `job id`, etc.